proposals/p0444.md

```diff
@@ -107,7 +107,7 @@ investment into making a copy.
 #### Caveats
 
 The
-[contributor wiki](https://forums.carbon-lang.dev/t/contributor-directory-wiki/134/2)
+[contributor wiki](https://github.com/carbon-language/carbon-lang/wiki/Contributor-directory)
```

Comment:
I'd suggest just marking the old link as broken. I think it's helpful to highlight that we previously had a contributor directory that wasn't where it is now.

Reply:
Okay. I've restored and annotated the broken link.